akka-persistence/src/main/scala/akka/persistence/journal/leveldb/LeveldbJournal.scala

```diff
@@ -33,13 +36,25 @@ private[persistence] class LeveldbJournal extends { val configPath = "akka.persi
  * INTERNAL API.
  */
 private[persistence] object LeveldbJournal {
+  sealed trait SubscriptionCommand
+
   /**
-   * Subscribe the `sender` to changes (append events) for a specific `persistenceId`.
-   * Used by query-side. The journal will send [[ChangedPersistenceId]] messages to
+   * Subscribe the `sender` to changes (appended events) for a specific `persistenceId`.
+   * Used by query-side. The journal will send [[EventAppended]] messages to
    * the subscriber when `asyncWriteMessages` has been called.
    */
-  case class SubscribePersistenceId(persistenceId: String)
-  case class ChangedPersistenceId(persistenceId: String) extends DeadLetterSuppression
+  case class SubscribePersistenceId(persistenceId: String) extends SubscriptionCommand
+  case class EventAppended(persistenceId: String) extends DeadLetterSuppression
+
+  /**
+   * Subscribe the `sender` to changes (appended events) for a specific `persistenceId`.
```

Comment:
First line here was copied from above message yet does not apply. We're listening for ids here, not events